app/controllers/api/openid_connect/authorizations_controller.rb

```diff
@@ -0,0 +1,120 @@
+module Api
+  module OpenidConnect
+    class AuthorizationsController < ApplicationController
```

Comment:
Assignment Branch Condition size for restore_request_parameters is too high. [18.03/15]